### ISSUE TITLE
Goroutine started on init() function

### ISSUE THREAD

**Issue by adrianlzt** [census-instrumentation/opencensus-go#1074]
**Describe the bug**
In this [init()](https://github.com/census-instrumentation/opencensus-go/blob/master/stats/view/worker.go#L27) function a gorouting is fired.

This looks like an strange behaviour for a lib, because any program importing this lib will have your goroutine running.
Found looking for a problem in Telegraf: https://github.com/influxdata/telegraf/pull/5612

**Comment by aarondl** [comment]
Found this goroutine running during my tests today. This strikes me as extremely bad practice to start unwanted goroutines no matter how "cheap" they are. If the package requires a Goroutine to run, starting it should be part of the public API that users have to start themselves.

**Comment by ncw** [comment]
Likewise importing these google APIs will also start the mystery go routine

- google.golang.org/api/drive/v2
- google.golang.org/api/drive/v3
- google.golang.org/api/storage/v1

**Comment by mholt** [comment]
I can't explain how, but I believe this goroutine has a serious bug causing the CPU to spin, reported in: #1200

**Comment by james-lawrence** [comment]
agreed this should not be happening. i had to investigate wtf the library was and whether it posed a security risk to my application.
